src/Simplex_tree/concept/SimplexTreeOptions.h

```diff
@@ -29,5 +29,7 @@ struct SimplexTreeOptions {
   static constexpr bool contiguous_vertices;
   /// If true, the lists of `Node` with same label are stored to enhance cofaces and stars access.
   static const bool link_nodes_by_label;
+  /// If true, Simplex_handle will not be invalidated after insertions or removals.
```

Comment:
Maybe we could mention here that it slows down most operations. Or maybe that's obvious, we wouldn't provide the false case if it did not have advantages. We don't mention that contiguous_vertices can speed up some operations, so **not** saying anything here either is consistent.

Reply:
That is what I thought. If there were no disadvantages to have stable simplex handles, it would not be at false by default. But we could add something if you think it is better, also for `contiguous_vertices` for consistency.